defaults/main.yml

```diff
@@ -94,6 +94,15 @@ telegraf__mqtt_consumer_topic: "default/#"
 telegraf__mqtt_consumer_id: "{{ inventory_hostname_short }}"
 telegraf__mqtt_consumer_ca: "/etc/telegraf/ca.pem"
 
+# logparser variables
+telegraf__logparser_configured: False
+telegraf__logparser_files: "[ '/var/log/nginx/welcome_access.log' ]"
+telegraf__logparser_read_from_beginning: false
+telegraf__logparser_watch_method: "inotify"
+telegraf__logparser_patterns: "[ '%{COMMON_LOG_FORMAT}' ]"
+telegraf__logparser_measurement_name: "nginx_access_log"
+telegraf__logparser_pattern_files: []
+telegraf__logparser_timezone: "Europe/Warsaw"
```

Comment:
To powinno być z konfiguracji roli debops.ntp albo pliku `/etc/timezone` (zmienianego przez tą rolę).

Reply:
Ewentualnie w grę wchodzi jest słówko `Local` tak jak w opisie parametru.